lib/exportOverlays.nix

```diff
@@ -44,27 +44,44 @@ let
         attrNames
         attrValues
         concatMap
+        concatStringsSep
         elem
         filter
         foldl'
         head
+        isAttrs
+        isFunction
         listToAttrs
         mapAttrs
+        tryEval
         ;
+
+      # Hopefully will fix a couple of edge cases. Even tho `lib` is defined as API here - it is not. Do not use it.
```

Comment:
Is this note meant for developers of exportOverlays? or users of `fakePkgs`

Reply:
I guess users if they decide to check the source